docs/tutorial/using-multiple-themes-together.md

```diff
@@ -2,4 +2,481 @@
 title: Using Multiple Themes Together
 ---
 
-_This tutorial is coming soon. In the meantime, check out the [introduction to using multiple themes in the docs](/docs/themes/using-multiple-gatsby-themes)._
+## What this tutorial covers
+
+- Composing multiple themes into a final site using [gatsby-theme-blog](/packages/gatsby-theme-blog/), [gatsby-theme-notes](/packages/gatsby-theme-notes/) and [gatsby-mdx-embed](https://gatsby-mdx-embed.netlify.com/) as examples
+- Component shadowing
+- Component shadowing for Theme-UI theme file
+
+## Prerequisites
+
+- [Gatsby fundamentals](/tutorial/#gatsby-fundamentals)
+- [What are Gatsby themes?](/docs/themes/what-are-gatsby-themes/)
+
+## Step 1: Create a new site using the hello world starter
+
+```shell
+gatsby new multiple-themes https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+## Step 2: Install the themes
+
+The first two themes we will compose together are [gatsby-theme-blog](/packages/gatsby-theme-blog/) and [gatsby-theme-notes](/packages/gatsby-theme-notes/).
+
+```shell
+npm install gatsby-theme-blog gatsby-theme-notes
+```
+
+## Step 3: Update gatsby-config.js
+
+The themes need to be added to the plugins array and we need some basic site metadata too.
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  siteMetadata: {
+    title: `Multiple Themes`,
+    description: `A tutorial for building a Gatsby site using multiple themes.`,
+    author: `Your name`,
+    social: [
+      {
+        name: `twitter`,
+        url: `https://twitter.com/gatsbyjs`,
+      },
+      {
+        name: `github`,
+        url: `https://github.com/gatsbyjs`,
+      },
+    ],
+  },
+  plugins: [
+    {
+      resolve: `gatsby-theme-blog`,
+      options: {
+        basePath: `/blog`,
+      },
+    },
+    {
+      resolve: `gatsby-theme-notes`,
+      options: {
+        basePath: `/notes`,
+      },
+    },
+  ],
+}
+```
+
+## Step 4: Run the site
+
+Next, run the site and see what you have:
+
+```shell
+cd multiple-themes
+gatsby develop
+```
+
+## Step 5: Add some content
+
+Not much to look at yet, just a plain `hello-world`, but it works! Behind the scenes our two themes [created content folders](/docs/themes/conventions/#initializing-required-directories) for us in the root directory of the site. We need to add some content to these folders.
```

Comment:
```suggestion
Behind the scenes, the two themes created content folders in the root directory of the site. In this step, you will add some content to these folders.
```
Try to avoid extra links when writing beginner-friendly tutorials as it might be a lot of distractions.